frontend/workflows/experimentation/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
+    "composite": true,
```

Comment:
This is needed due to some weird JS compilation issues - we use the same attribute in other shared packages such as clutch core too so I think it should be fine